ui/app/components/clients/current.js

```diff
@@ -76,13 +97,46 @@ export default class Current extends Component {
       .mounts?.find((mount) => mount.label === auth);
   }
 
-  get countsIncludeOlderData() {
-    if (!this.latestUpgradeData) {
-      return false;
+  get upgradeDuringCurrentMonth() {
+    if (!this.upgradeVersionHistory) {
+      return null;
+    }
+    const upgradesWithinData = this.upgradeVersionHistory.filter((upgrade) => {
+      // TODO how do timezones affect this?
+      let upgradeDate = new Date(upgrade.timestampInstalled);
+      return isAfter(upgradeDate, startOfMonth(new Date()));
+    });
+    // return all upgrades that happened within date range of queried activity
+    return upgradesWithinData.length === 0 ? null : upgradesWithinData;
+  }
+
+  get upgradeVersionAndDate() {
+    if (!this.upgradeDuringCurrentMonth) {
+      return null;
+    }
+    if (this.upgradeDuringCurrentMonth.length === 2) {
+      return 'Vault was upgraded to 1.9 and 1.10 during this month.';
```

Comment:
For future-proofing, could we use the value dates here too? That would even allow us to remove the length check, so for example:
```js
let versions = this.upgradeDuringCurrentMonth.map(upgrade => upgrade.id).join(' and ');
return `Vault was upgraded to ${versions} during this month`;
```